openshift/job-template.yaml

```diff
@@ -0,0 +1,144 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: package-analyzer
+  annotations:
+    description: "Thoth: Package Analyzer Template"
+    openshift.io/display-name: "Thoth: Package Analyzer Template"
+    version: 0.4.0
+    tags: thoth,ai-stacks,aistacks,package-analyzer
+    template.openshift.io/documentation-url: https://github.com/Thoth-Station/
+    template.openshift.io/long-description: >
+      This template defines resources needed to run package analyzer in Thoth to OpenShift, it is used
+      to create new OpenShift Jobs running the package analyzer.
+    template.openshift.io/provider-display-name: Red Hat, Inc.
+  labels:
+    template: package-analyzer
+    app: thoth
+    component: package-analyzer
+
+objects:
+  - apiVersion: batch/v1
+    kind: Job
+    metadata:
+      name: package-analyzer
+      labels:
+        app: thoth
+        component: package-analyzer
+        operator: graph-sync
```

Comment:
I copied from package-extract, is it OK to put it here?

Reply:
Yes, it will not work now as you haven't implemented sync functionality yet, but this is the right step to trigger it.

Reply:
Thanks